cumulusci.yml

```diff
@@ -399,6 +406,9 @@ flows:
                 task: deploy_dev_config
             7:
                 flow: eda_settings
+            8:
+                task: deploy_release_gating
+                when: "'hed' not in org_config.installed_packages"
```

Comment:
Please remove this after incorporating the other changes. This will cause the other metadata type to deploy to our dev environments so we can see the appropriate failures.
```suggestion
```

Please also add the `deploy_release_gating` task to the `config_qa` flow so that QA can also see this applied.